test/src/pages/adoption/how-using.test.js

```diff
@@ -1,71 +1,29 @@
+import {makeMountRender} from '../../../helpers/jsx-test-utils.jsx';
 import HowUsing from '~/pages/adoption/how-using/how-using';
-import instanceReady from '../../../helpers/instance-ready';
-import {clickElement, doInput} from '../../../test-utils';
 
 describe('HowUsing', () => {
-    let props, instance, ready, usingInfo;
 
-    beforeEach(() => {
-        props = {
+    let wrapper;
+
+    beforeEach((done) => {
+        const props = {
             selectedBooks: [],
-            disable: false
+            selectedBooks: [{
+                text: 'First Book',
+                value: 'first-book'
+            }]
         };
-        ({instance, ready} = instanceReady(
-            HowUsing,
-            () => props,
-            (newValue) => {
-                usingInfo = newValue;
-            }
-        ));
+        wrapper = makeMountRender(HowUsing, props)();
+        setTimeout(() => {
+            wrapper.update();
+            done();
+        }, 10);
```

Comment:
you can return a promise in `beforeEach` and it'll wait for it to resolve before running specs

Reply:
That's what `done` is for: you call it to resolve a promise that `beforeEach` creates for you. The timeout was to allow an internal-to-the-component async event to happen. But I discovered I could move the `update` call out of `beforeEach` and into the test and get the same effect.